### PR TITLE
typescript-async-await - Tré Mitchell

### PR DESCRIPTION
**Pull request by TreMitchell**

### CODE REVIEW

File: typescript-async-await/await.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  console.log(elapsed(), 'readChained1:', msg1Chained);
+  const msg2Chained = await read(msg1Chained);
+  console.log(elapsed(), 'readChained2:', msg2Chained);
+  const msg3Chained = await read(msg2Chained);
+  console.log(elapsed(), 'readChained3:', msg3Chained);
+}
+
+readOnce()
+  .then(() => readSeveral())
+  .then(() => readChained());
```
Comment (shawnkost) [inline typescript-async-await/await.ts]: These should be done using `await`

File: typescript-async-await/exceptions.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    const msg3Chained = await read(msg2Chained);
+    console.log(elapsed(), 'throwChained3:', msg3Chained);
+  } catch (error) {
+    console.log(elapsed(), 'throwChained Error:', error);
+  }
+}
+
+throwOnce()
+  .then(() => throwSeveral())
+  .then(() => throwChained());
```
Comment (shawnkost) [inline typescript-async-await/exceptions.ts]: These should be done using `await`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple small changes here!

File: typescript-async-await/await.ts (hunk 1 above)
Comment (TreMitchell) [reply typescript-async-await/await.ts]: I've made the changes to both of the files

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**